src/test/java/automation/api/ApiTest.java

```diff
@@ -0,0 +1,136 @@
+package automation.api;
+
+import automation.app.*;
+import automation.base.BaseTestNG;
+import automation.controller.BookingController;
+import io.restassured.response.ValidatableResponse;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+import static automation.Config.*;
+import static io.restassured.RestAssured.*;
+public class ApiTest extends BaseTestNG {
+    @Test
+    public void addNewBookingTest() {
+
+        Booking booking = Booking.newBuilder()
+                .setFirstname("Johnny")
+                .setLastname("Depp")
+                .setTotalprice(222)
+                .setDepositpaid(true)
+                .setBookingdates(Bookingdates.builder().checkin("2018-01-01")
+                        .checkout("2019-02-02").build())
+                .setAdditionalneeds("Latte")
+                .build();
+
+        var response = given()
+                .spec(REQ_SPEC)
+                .header("Authorization", "Bearer " + getToken())
```

Comment:
Is this working? token should be in cookie header not an authorization bearer token